src/components/TodoHeader.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useRef, useState } from 'react';
+import cn from 'classnames';
+import { ErrorMessage } from '../types/ErrorMessage';
+
+type Props = {
+  error: ErrorMessage;
+  isToogleAll: boolean | null;
```

Comment:
In what case should it be null? it's a bit confusing

Reply:
in this case boolean defines ` 'active'` button or not, but `null ` defines is the button in the input at all or not